Mathlib/CategoryTheory/Limits/Shapes/Images.lean

```diff
@@ -178,17 +179,18 @@ variable {f}
 
 /-- Data exhibiting that a given factorisation through a mono is initial. -/
 structure IsImage (F : MonoFactorisation f) where
-  lift : ∀ F' : MonoFactorisation f, F.i ⟶ F'.i
-  lift_fac' : ∀ F' : MonoFactorisation f, lift F' ≫ F'.m = F.m := by obviously
+  /-- The "image" of `F` maps to any other "image" -/
```

Comment:
Please remove the docs.

Reply:
Added `@[inherit_doc]` to all the fields